exercises/practice/kindergarten-garden/.docs/instructions.append.md

```diff
@@ -1,4 +1,5 @@
 # Instructions append
 
-In this exercise you're going to be using a concept called dynamic programming. https://en.wikipedia.org/wiki/Dynamic_programming
-Your solution could use the class method `#define_singleton_method` or the combination of `#method_missing` and `#respond_to_missing`.
+The exercise requires you to write a class that provides a single method for each child in the kindergarten, respectively.
+You can make your work easier and less repetitive if, instead of manually defining 12 different methods, you use a bit of metaprogramming.
+Your solution could use the method `#define_method` (or `#define_singleton_method`), or alternatively a combination of `#method_missing` and `#respond_to_missing?`.
```

Comment:
```suggestion
If you're not sure where to start, explore the [`method_missing`](https://www.leighhalliday.com/ruby-metaprogramming-method-missing) or [`define_method`](http://rubymonk.com/learning/books/2-metaprogramming-ruby/chapters/25-dynamic-methods/lessons/72-define-method) methods.
```

Reply:
I have no problem with the rephrasing, but I kinda dislike the fact that the `method_missing` article overrides `respond_to?` (and only relatively late in the text) instead of `respond_to_missing?`, which would be more safe and idiomatic, as I am lead to believe (although I am by no means an expert on Ruby).

Problem is, I cannot find an article that would explain it more (only those that already presume the reader knows what `method_missing` is, e.g. https://thoughtbot.com/blog/always-define-respond-to-missing-when-overriding).

Reply:
I still consider `respond_to_missing?` to be the new kid on the block, although you're very right :) 

OK, Let's stick with these links for now and as soon as we have better Concept pages on Exercism for these in Ruby, we'll switch to them.

@meatball133 I don't find the Ruby docs to be clear enough for Ruby newbies really. I'd rather like to an explanation tutorial/blog (ideally our concept pages!)